sys/include/net/gnrc/netdev2/xbee_adpt.h

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (C) 2016 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @addtogroup  net_gnrc
+ * @{
+ *
+ * @file
+ * @brief       GNRC to XBee netdev2 glue code interface
+ *
+ * The XBee devices use their own proprietary format for communicating between
+ * the host CPU and the device over UART. The XBee device driver expects the
+ * data to send to be given in this format, hence we need to introduce an XBee
+ * specific adaption layer, that translates between GNRC and the XBee
+ * proprietary header format.
```

Comment:
Sorry for being nit-picky, but some reference to the IEEE 802.15.4 glue code is required here. The current text reads like xbee has nothing to do with IEEE 802.15.4 at all.